modules/ROOT/pages/understanding-editor-loads.adoc

```diff
@@ -0,0 +1,65 @@
+= Understanding editor loads
+:navtitle: Understanding editor loads
+:description: Relevant information for Tiny Cloud users to help understand editor loads for {productname}.
+:keywords: invalid-api-key, API, {productname}, cloud, frequently asked questions
+
+== Understanding editor loads for {productname}
+
+[IMPORTANT]
+This information is only relevant to Tiny Cloud users. Users who self-host the open source version of {productname} are not subject to editor load restrictions, but must comply with the ////<Insert link>[open source license].//// 
```

Comment:
Need to update/add link prior to merging.

Reply:
This should always point to the released license, no matter which one we choose
`https://github.com/tinymce/tinymce/blob/master/LICENSE.TXT`

Reply:
```suggestion
This information is only relevant to Tiny Cloud users. Users who self-host the open source version of {productname} are not subject to editor load restrictions, but must comply with the https://github.com/tinymce/tinymce/blob/master/LICENSE.TXT[open source license]. 
```